CHANGELOG.md

```diff
@@ -11,4 +11,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - A changelog file.
 
+### Fixed
+
+- An issue where Alice would not verify if Bob's Bitcoin lock transaction is semantically correct, i.e. pays the agreed upon amount to an output owned by both of them.
```

Comment:
Could be worth mentioning here that this is a breaking change on the network layer!

Note: Could mark it as `Security` instead of `Fixed` (https://keepachangelog.com/en/1.0.0/)